config/default.toml

```diff
@@ -55,3 +50,14 @@
     dsn                   = "dummy"
     enabled               = false
     logLevel              = "Error"
+
+[services]
```

Comment:
TODO: Think if this nesting is required or not.

Reply:
i have 2 things in mind, i am confused on 2 structures (will do in other PR to keep this one short), also need to merge all dev, docker and normal one into one config, all this will be handlled in config related changes PR

current thoughts
1.
```
[producer]
a=b
a1=b1
[push-consumer]
a=b
a2=b2
[pull-consumer]
a=b
a3=b3
```
this will completely separate out the configs for each service, we can have separate files itself then, there will be duplicates of settings here but easy to make changes with respect to 1 service without impacting others

2.
[common]
a=b
[producer]
a1=b1
[push-consumer]
a2=b2
[pull-consumer]
a3=b3

merging configs can be pain here but still doable